src/core/symbology/qgsmarkersymbollayer.cpp

```diff
@@ -3598,6 +3615,8 @@ QRectF QgsFontMarkerSymbolLayer::bounds( QPointF point, QgsSymbolRenderContext &
   {
     chrWidth *= scaledSize / mOrigSize;
   }
+  if ( !qgsDoubleNear( mFontSizeScale, 1.0 ) )
```

Comment:
nitpicking: doing this test is probably more expensive than always doing the multiplication

Reply:
> nitpicking

That's the best type of picking! Fixed.